derive/src/pymodule.rs

```diff
@@ -173,33 +173,22 @@ impl Module {
                 }
                 "pyattr" => {
                     attr_idxs.push(i);
-                    match item.typ {
-                        ItemType::Fn => {
-                            items.push((
-                                Self::extract_evaluated_attr(item.ident, into_nested()?)?,
-                                meta_span,
-                            ));
-                        }
-                        ItemType::Const => {
-                            items.push((
-                                Self::extract_const_attr(item.ident, into_nested()?)?,
-                                meta_span,
-                            ));
-                        }
-                        ItemType::Struct => {
+                    let item = match item.typ {
+                        ItemType::Fn => Self::extract_evaluated_attr(item.ident, into_nested()?)?,
```

Comment:
You could probably just do `Const | Use => ...` here

Reply:
i will fix this one in the next PR